man/udev.xml

```diff
@@ -437,6 +437,8 @@
             <para>Note that running programs that access the network or mount/unmount
             filesystems is not allowed inside of udev rules, due to the default sandbox
             that is enforced on <filename>systemd-udevd.service</filename>.</para>
+            <para>Please not that <literal>:=</literal> and <literal>:</literal> are clearing
```

Comment:
I think `<literal>:</literal>` should be `<literal>=</literal>`.

Reply:
You are right, for sure. There was also another typo :)